python_modules/libraries/dagster-powerbi/dagster_powerbi_tests/test_asset_specs.py

```diff
@@ -135,14 +153,32 @@ def test_refreshable_semantic_model(
     assert result.success is success
 
 
-def test_using_cacheable_assets_defs(workspace_data_api_mocks: responses.RequestsMock) -> None:
```

Comment:
renamed to `test_state_driven_defs` (no longer uses cacheable assets defs anyway)